data_structures/src/chain.rs

```diff
@@ -1087,30 +1088,25 @@ impl DataRequestOutput {
     /// Calculate the total value of a data request, return error on overflow
     ///
     /// ```ignore
-    /// total_value = (witness_reward + commit_fee + reveal_fee) * witnesses + tally_fee
+    /// total_value = (witness_reward + commit_and_reveal_fee + commit_and_reveal_fee) * witnesses
```

Comment:
why is it counted twice?

Reply:
Ahh got it, I thought for a moment that commit_and_reveal_fee was the sum of both fees

Reply:
its the same for both, so now we have only one field to represents the 2 fees